### ISSUE TITLE
Brainstorm potential improvements to R-project.org

### ISSUE THREAD

**Issue by hadley** [ropensci/unconf#26]
I'd love to find out what people think are the biggest problems with the existing site, so I can prioritise work over the next year.

**Comment by karthik** [comment]
I have loads of ideas on this and look forward to chatting in person.

**Comment by gaborcsardi** [comment]
:+1: 

In general: tools for better package discovery:
- package search (search for packages, search package code, search package docs, etc.)
- package top lists: most downloads, most reverse deps, most recently updated, etc.

Blog by r-core about the current development directions. If they want to write it. Maybe they are not really into blogging, but it would be great to read about what they are planning to work on, so it might be worth asking.

**Comment by hadley** [comment]
@gaborcsardi unfortunately that's all out of scope - I can only affect the main page, not CRAN. Anything that's dynamic is going to be particularly hard.

**Comment by karthik** [comment]
> package search (search for packages, search package code, search package docs, etc.)

The functionality of [RDocumentation](http://www.rdocumentation.org/) is impressive this way. 
Also linking in discoverability of new content (see #9)

**Comment by eddelbuettel** [comment]
The format is basically given:  markdown through pandoc to static html. So nothing dynamic or Javascript-y.  We basically share the same constraint as the (static, easy for mirrors) CRAN pages.

**Comment by hadley** [comment]
Another avenue for that sort of stuff is crantastic - http://crantastic.org

**Comment by benmarwick** [comment]
How about feeding rstudio cran mirror download stats to crantastic for package ranking?

**Comment by gmbecker** [comment]
@benmarwick I would worry about those numbers being pretty nonrepresentative in general. People who use rstudio and those that don't are likely to be dissimilar in important ways.

**Comment by hadley** [comment]
@gmbecker I suspect that's true, but I also suspect (although I have no data to back it up), the RStudio mirror is responsible for serving >80% of CRAN packages

**Comment by timphan** [comment]
I love R graphics, and it'd great if the page could have some beautiful ggplot/rCharts/etc graphs to illustrate the potential of R's visualizations. I find that it's usually a good introduction for someone who isn't a programmer or statistician (we can all share a taste for Excel Chart Wizard, right?).

**Comment by eddelbuettel** [comment]
That is planned in possibly a carousel at the front-page but it may be better to allow some other dust to settle and to get some changes in first such as CRAN conversion.

**Comment by jhollist** [comment]
Not sure if this counts just as an improvement to the site, but the only method to become a supporting member is to fax or mail a hard copy form...  It would be nice to sign up and  use a CC to pay online.

Would probably increase membership significantly.

**Comment by hadley** [comment]
@jhollist yeah, that's a somewhat separate project. I can give you an update in person

**Comment by jhollist** [comment]
Thanks @hadley.  It makes sense that it is a separate project.  Unfortunately I won't be at the unconf so won't be able to get the update.  I can get the update when one is generally available.  Good enough to know that it sounds like it is on the Foundation's radar.
